app/controllers/group_assignments_controller.rb

```diff
@@ -21,4 +21,35 @@ def create
       end
     end
   end
+
+  def destroy
+    @meeting = Meeting.find(params["meeting_id"])
+    @individual = Individual.find(params["individual_id"])
+
+    if params["group_assignment"]["group"].empty?
+      flash[:alert] = "Please select a valid group!"
+      redirect_to meeting_individual_path(@meeting, @individual)
+    else
+
+      @group = Group.find(params["group_assignment"]["group"])
+      @group_assignment = GroupAssignment.where(group_id: @group.id, individual_id: @individual.id)[0]
```

Comment:
stylistically, most Rails devs will use a `.first` invocation.

This is because, you will otherwise, unnecessarily retrieve all GroupAssignments when you just want to fetch the first entry. This performs better as it relies on SQL to constrain your query to only retrieve the first result.